docs/index.rst

```diff
@@ -9,55 +9,100 @@ If you want to build a completely different site with all the same Django
 optimizations for security, scalability, L10n, and ease of use, check out
 Mozilla's `Playdoh starter kit <http://playdoh.readthedocs.org/>`_.
 
-
 .. _install-with-docker:
 
 Quickstart
 ----------
+Want the easiest way to start contributing to AMO? Try our docker-based
+development environment.
```

Comment:
Do we want to say that this is the preferred, standard (and only supported?) way to do local dev? ping @andymckay